cumulus_library/cli.py

```diff
@@ -113,73 +113,126 @@ def export_all(self, study_dict: Dict):
             self.export_study(study_dict[key])
 
 
-def get_study_dict() -> Dict[str, PosixPath]:
+def get_abs_posix_path(path: str) -> PosixPath:
+    """Conveince method for hanlding abs vs rel paths"""
+    if path[0] == "/":
+        return Path(path)
+    else:
+        return Path(Path.cwd(), path)
+
+
+def create_template(path: str) -> None:
+    """Creates a manifest in target dir if one doesn't exist"""
+    abs_path = get_abs_posix_path(path)
+    manifest_path = Path(abs_path, "manifest.toml")
+    if manifest_path.exists():
+        sys.exit(f"A manifest.toml already exists at {abs_path}, skipping creation")
+    template_path = Path(
+        Path(__file__).resolve().parents[0], "studies/template/manifest.toml"
+    )
+    abs_path.mkdir(parents=True, exist_ok=True)
+    manifest_path.write_bytes(template_path.read_bytes())
+
+
+def get_study_dict(alt_paths: List) -> Optional[Dict[str, PosixPath]]:
     """Convenience function for getting directories in ./studies/
 
     :returns: A list of pathlib.PosixPath objects
     """
     manifest_studies = {}
-    library_path = Path(__file__).resolve().parents[0]
-    for path in Path(f"{library_path}/studies").iterdir():
-        if path.is_dir():
-            manifest_studies[path.name] = path
+    paths = [Path(Path(__file__).resolve().parents[0], "studies")]
+    if alt_paths is not None:
```

Comment:
nit: eh let em error out if they break your type hinting of a required arg  :smile:

Reply:
well this wass more 'did you even supply one'? so that way i can rely on list typing for addition. though i guess i could make the default `[]` instead of None and not worry about it?

Reply:
There is no default right now, you have to supply the argument. And you type hinted it as a List, not Optional[List], so the hinting suggests you need to pass a real list in, no None allowed.